pkg/controller/cainjection/secret_controller.go

```diff
@@ -58,6 +58,12 @@ func addSecretReconciler(mgr manager.Manager, r reconcile.Reconciler) error {
 			if oldSecretDest != newSecretDest {
 				return true
 			}
+
```

Comment:
why did you have to change this file? from what I understand you were just adding a new controller...

Reply:
It looks like stuff from when I thought I had to go that direction made it on the wrong branch let me clean it up